.github/workflows/create-newsletter.yaml

```diff
@@ -0,0 +1,45 @@
+name: Create Newsletter
+on:
```

Comment:
Does that run on every branches ?

Reply:
Good point, let me fix that :)

Reply:
Made sure it only pulls and pushed the `source` branch.